### DISCUSSION TITLE
Assert if a string is a valid finite numeric value

### DISCUSSION THREAD

**Discussion by romanstetsyk** [stdlib-js/stdlib#1015] [stdlib-bot Q&A]
Is there a package to check if a string is a valid finite numeric value?

[a comment by an automated account is not shown]

**Reply by kgryte** [reply]
@romanstetsyk If you are just wanting to know that a value is finite, you can do something like

```javascript
const isFinite = require( '@stdlib/assert-is-finite' ).isPrimitive;

// ...

const bool = isFinite( parseFloat( value ) );
// returns <boolean>
```

If you are wanting to know if a value is within a certain range, you can do something like

```javascript
const isBetween = require( '@stdlib/assert-is-between' );

// ...

const high = 100;
const low = 10;

const bool = isBetween( parseFloat( value ), low, high );
// returns <boolean>
```

## Links

- [`@stdlib/assert-is-finite`](https://github.com/stdlib-js/assert-is-finite)
- [`@stdlib/assert-is-between`](https://github.com/stdlib-js/assert-is-between)
